java/app/build.gradle

```diff
@@ -5,9 +5,18 @@ Properties properties = new Properties()
 properties.load(project.rootProject.file('local.properties').newDataInputStream())
 
 fullstory {
-    org properties.getProperty('org') // your ORG_ID
-    enabledVariants properties.getProperty('enabledVariants') // enable variants that you want FS to instrument (all)
-    server properties.getProperty('server') // optional
+    // ORG INFO
+    if(properties.getProperty('org') == null) throw new Exception('ORG ID is null, please configure your orgnization information in local.properties')
+    org properties.getProperty('org') // your ORG_ID in local.properties, or set it here directly
```

Comment:
> // your ORG_ID in local.properties, or set it here directly

Since you're throwing an error in the line above if not set in `local.properties` doesn't this mean that it can only be set in `local.properties` and can't be set directly?

Reply:
Ah! good catch!